index.js

```diff
@@ -0,0 +1,125 @@
+const URL = 'https://trektravel.herokuapp.com/trips';
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p>`
+  content += "<ul>";
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const loadTrips = () => {
+  reportStatus('Loading trips...');
+  
+  axios.get(URL)
+    .then((response) => {
+      reportStatus(`Successfully loaded ${response.data.length} trips`);
+      const trips = response.data;
+      displayTripList(trips);
```

Comment:
I like that you broke `displayTripList out as a separate function - good organization!